code-challenges.test.js

```diff
@@ -1,5 +1,10 @@
 // ASSESSMENT 2: Coding Practical Questions with Jest
 
+const { assertTaggedTemplateExpression } = require("@babel/types")
+const { string } = require("yargs")
+
```

Comment:
Sometimes these imports cause errors when you run your tests. If that ever becomes an issue you can remove them.